packages/ios_platform_images/lib/ios_platform_images.dart

```diff
@@ -138,6 +138,74 @@ class IosPlatformImages {
     return _FutureMemoryImage(bytesCompleter.future, scaleCompleter.future);
   }
 
+  /// Loads an image from the system.  The equivalent would be:
+  /// `[UIImage systemImageNamed:name]`.
+  ///
+  /// Throws an exception if the image can't be found.
+  ///
+  /// [colors] takes a list of colors to be applied to the corresponding layer
```

Comment:
nit (here and elsewhere)L According to flutter style docs should be grammatically correct and therefore this sentence should start with a capital letter (may have to rephrase it a bit to not have it start with [color]).